code/game/objects/structures/dresser.dm

```diff
@@ -56,12 +56,13 @@
 
 /obj/structure/dresser/crowbar_act(mob/user, obj/item/I)
 	. = TRUE
-	if(!I.tool_start_check(src, user, 0))
+	if(!I.use_tool(src, user, 0))
 		return
 	TOOL_ATTEMPT_DISMANTLE_MESSAGE
 	if(I.use_tool(src, user, 50, volume = I.tool_volume))
 		TOOL_DISMANTLE_SUCCESS_MESSAGE
-
+		new /obj/item/stack/sheet/wood(loc, 30)
```

Comment:
Best to override `deconstruct(..)` and call it here.
Same for the others

Reply:
Forgot about that too!

Reply:
So just `deconstruct(disassembled = TRUE)` there?

Reply:
It ended up being more complicated than I thought, so I'm pretty sure I did the opposite of what you suggested.
Either way, it still seems to work.